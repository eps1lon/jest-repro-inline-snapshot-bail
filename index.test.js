test("DOES NOT bail on toMatchInlineSnapshot()", () => {
  expect(1).toMatchInlineSnapshot(`2`);
  expect(3).toMatchInlineSnapshot(`4`);
});

test("DOES NOT bails on toMatchSnapshot()", () => {
  expect(1).toMatchSnapshot();
  expect(3).toMatchSnapshot();
});

test("bails on toEqual()", () => {
  expect(1).toEqual(2);
  expect(3).toEqual(4);
});
