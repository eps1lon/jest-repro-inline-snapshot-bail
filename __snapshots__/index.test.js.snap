// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOES NOT bails on toMatchSnapshot() 1`] = `1`;

exports[`DOES NOT bails on toMatchSnapshot() 2`] = `3`;
